shell/openxr/mobile/java/com/facebook/igl/shell/openxr/gles/AndroidManifest.xml

```diff
@@ -1,42 +1,91 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.facebook.igl.shell.openxr.gles"
-          android:versionCode="1"
-          android:versionName="1.0"
-          android:installLocation="auto">
-  <uses-sdk android:minSdkVersion="24" android:targetSdkVersion="25" />
-  <uses-feature android:glEsVersion="0x00030000"/>
-  <uses-feature android:name="com.oculus.experimental.enabled" android:required="false" />
+    package="com.facebook.igl.shell.openxr.gles"
+    android:versionCode="1"
+    android:versionName="1.0"
+    android:installLocation="auto">
+
+  <uses-feature android:name="android.hardware.usb.host" />
+
+  <uses-feature android:name="android.hardware.vr.headtracking" android:required="true" android:version="1" />
+
   <uses-feature android:name="com.oculus.feature.PASSTHROUGH" android:required="false" />
-  <uses-feature android:name="oculus.software.handtracking" android:required="false" />
+  <uses-feature android:name="android.hardware.microphone" android:required="false" />
 
-  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.RECORD_AUDIO" />
+  <uses-permission android:name="android.permission.BLUETOOTH" />
+  <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
```

Comment:
I need record audio for the microphone support for PC VR streaming purposes, I have implemented full OBOE support in my branch. Do you want this? I could add it to the base IGL sample apps instead of here. Otherwise I can remove these. Bluetooth is needed for gamepads and earphones I believe. BT admin is for pairing so probably out of scope. Network state is to check for wifi/internet connectivity.